projects/fusion-e2e/tests/components/alert/alert-component.ts

```diff
@@ -1,45 +1,59 @@
-import {Page} from '@playwright/test';
 import {AlertTestIdModifiers} from '@ironsource/fusion-ui/entities';
 import {getTestId, getTestIdSelector} from '../../global/utils';
-
-export class AlertComponent {
-    readonly page: Page;
-
-    constructor(page: Page) {
-        this.page = page;
+import {BaseComponent} from '../base-component';
+import {BaseElement} from '../../behavior';
+import {ButtonComponent} from '../button/button-component';
+
+export class AlertComponent extends BaseComponent {
+    private wrapperElement: BaseElement;
```

Comment:
Just a naming suggestion, but you don’t have to state 'Element' or 'Component' at the end of everything. For example actionButtonElement can be just actionButton and AlertComponent can be Alert

Reply:
It is OK also like this